<!-- -->
<launch>
    <node pkg="adeye"
          type="ExperimentB.py"
          name="ExperimentB"
          output="screen"/>
</launch>
